server/src/main/java/org/cloudfoundry/identity/uaa/scim/vmidentity/VmidentityScimGroupExternalMembershipManager.java

```diff
@@ -72,4 +72,22 @@ public void unmapAll(String groupId) throws ScimResourceNotFoundException {
 
     }
 
+    @Override
+    public List<ScimGroupExternalMember> query(String filter) {
+        // TODO Auto-generated method stub
```

Comment:
why are we leaving unimplemented?
i suggest:
1. let's add a comment as to why this is ok. (and remove autho generated comment, as this way it just looks like unfinished code)
2. it is likely safer to make seach ones return empty arrays, instead of nulls( with arrays callers usually not expect nulls).
thanks.